Homeworks/Homework SQL/SQLQuery-Homework.sql

```diff
@@ -0,0 +1,339 @@
+ALTER TABLE Books
```

Comment:
For some reason I can't add a comment on the png file so I'm adding some notes here:
- it's great that you decided to build this diagram even if I didn't specifically ask for it - I'm sure you realized it is really useful and helps you to understand the database schema easier.
- the Books_Sales table is not represented as it should be because it links the Books and Sales tables and not the Books and Libraries table as you represented it in the diagram.